ansible/roles/zsh/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+zsh__config_path: "/home/{{ zsh__user }}"
+zsh__config_owner: "{{ zsh__user }}"
+zsh__config_group: "{{ zsh__user }}"
```

Comment:
It would be nice to have docs for these.  Also, `zsh__user` doesn't seem to be documented or referenced anywhere.